### PR TITLE
The NuGet was missing the reference to the Support v4 Nuget.

### PR DESCRIPTION
**Pull request by topgenorth**
This replaces PR #158 - Updated the README's for the binding.

### CODE REVIEW

**Review by bholmes — APPROVED** [review]
LGTM

File: Android/FirebaseJobDispatcher/nuget/Xamarin.Firebase.JobDispatcher.nuspec [hunk 1]
```diff
@@ -11,6 +11,9 @@
     <copyright>Copyright (c) 2017 Xamarin Inc</copyright>
     <projectUrl>https://github.com/xamarin/XamarinComponents/tree/master/Android/FirebaseJobDispatcher</projectUrl>
     <licenseUrl>https://github.com/xamarin/XamarinComponents/blob/master/Android/FirebaseJobDispatcher/License.md</licenseUrl>
+    <dependencies>
+        <dependency id="Xamarin.Android.Support.v4" version="25.3.1" />
```
Comment (Redth) [inline Android/FirebaseJobDispatcher/nuget/Xamarin.Firebase.JobDispatcher.nuspec:15]: Shouldn't `Xamarin.GooglePlayServices.Gcm` be listed as a dependency too? or is it not strictly _required_

**Comment by topgenorth** [comment]
So AFAIK, Google Play Services has to be installed on the device, but GCM itself is not a dependency of the library (the Firebase Job Dispatcher docs are kind of hazy on this).  It's kind of weird: You can install an app that uses the FJD on a device without Google Play Services and it will run - however scheduling jobs silently fails (or that is what happens in my tests).

**Comment by Redth** [comment]
So, curious why you have them as a reference on the project?

**Comment by topgenorth** [comment]
So, it seems that the binding will work without having the GCM reference (at least it did on my device). The existing GCM references were a left-over from when I first started on the binding and GCM was necessary. 

I've updated the binding project by removing GCM, and consolidated to a single solution (instead of one for the sample and one for the binding). I'll open another PR for those changes, I believe we can close this issue off.

**Pull request merged**